runtest.py

```diff
@@ -240,9 +251,12 @@ def __getattr__(self, attr):
 sys.stdout = Unbuffered(sys.stdout)
 sys.stderr = Unbuffered(sys.stderr)
 
+# possible alternative: switch to using print, and:
+# print = functools.partial(print, flush)
+
 if options.output:
     logfile = open(options.output, 'w')
-    class Tee(object):
+    class Tee():
```

Comment:
no object parent class?

Reply:
in Py3, that's all classes, you don't need to list.